Gruntfile.js

```diff
@@ -23,9 +23,7 @@ module.exports = function(grunt) {
       './examples/shared/*.js',
       '.module-cache',
     ],
-    /*eslint-disable camelcase */
-    compare_size: require('./grunt/config/compare_size'),
-    /*eslint-enable camelcase */
+    'compare-size': require('./grunt/config/compare-size'),
```

Comment:
I don't think you can do this since the actual task is `compare_size` and the way grunt works is that it matches the config name up with the task of the same name. Does the task still work for you?